json_diff_test.go

```diff
@@ -1,10 +1,8 @@
-package jsondiff_test
+package jsondiff
```

Comment:
Is there any reason to change the test package?

Reply:
not a good reason. I was having some problems running the tests. Just fixed it, thanks for noticing :)